Generation/src/components/MomentumRangeParticleGun.cpp

```diff
@@ -106,35 +106,29 @@ void MomentumRangeParticleGun::generateParticle(Gaudi::LorentzVector& momentum,
 }
 
 StatusCode MomentumRangeParticleGun::getNextEvent(HepMC::GenEvent& theEvent) {
-  Gaudi::LorentzVector theFourMomentum ;
-  Gaudi::LorentzVector origin ;
+  Gaudi::LorentzVector theFourMomentum;
+  Gaudi::LorentzVector origin;
   // note: pgdid is set in function generateParticle
-  int thePdgId ;
-  generateParticle( theFourMomentum , origin , thePdgId );
+  int thePdgId;
+  generateParticle(theFourMomentum, origin, thePdgId);
 
   // create HepMC Vertex --
   // by calling add_vertex(), the hepmc event is given ownership
   //  of the vertex
-  HepMC::GenVertex * v = new HepMC::GenVertex( HepMC::FourVector( origin.X() ,
-								  origin.Y() ,
-								  origin.Z() ,
-								  origin.T() ) ) ;
+  HepMC::GenVertex* v = new HepMC::GenVertex(HepMC::FourVector(origin.X(), origin.Y(), origin.Z(), origin.T()));
   // create HepMC particle --
   // by calling add_particle_out(), the hepmc vertex is given ownership
   // of the particle
-  HepMC::GenParticle * p = new HepMC::GenParticle( HepMC::FourVector( theFourMomentum.Px() ,
-								      theFourMomentum.Py() ,
-								      theFourMomentum.Pz() ,
-								      theFourMomentum.E()  ) ,
-						   thePdgId ,
-						   1 ) ; // hepmc status code for final state particle
+  HepMC::GenParticle* p = new HepMC::GenParticle(
+      HepMC::FourVector(theFourMomentum.Px(), theFourMomentum.Py(), theFourMomentum.Pz(), theFourMomentum.E()),
+      thePdgId,
+      1);  // hepmc status code for final state particle
 
-  v -> add_particle_out( p ) ;
+  v->add_particle_out(p);
 
-  theEvent.add_vertex( v ) ;
-  theEvent.set_signal_process_id( 0 ) ;
-  theEvent.set_signal_process_vertex( v ) ;
+  theEvent.add_vertex(v);
+  theEvent.set_signal_process_id(0);
```

Comment:
Good point. I think 0 is a good value as any for a particle gun, but for a default-constructed HepMC::GenEvent it is 0 anyway, so I think we can remove this line, just for the rare case in which the gun is used together with something else which sets it.